disable qcncodecs, qjpcodecs, qkrcodecs, qtwcodecs codecs, not sure how to use them from system qt4

--- phantomjs-1.9.7/./src/main.cpp~	2014-03-02 16:26:44.000000000 +0200
+++ phantomjs-1.9.7/./src/main.cpp	2014-03-02 16:27:37.049043803 +0200
@@ -46,11 +46,6 @@
 
 #if !defined(QT_SHARED) && !defined(QT_DLL)
 #include <QtPlugin>
-
-Q_IMPORT_PLUGIN(qcncodecs)
-Q_IMPORT_PLUGIN(qjpcodecs)
-Q_IMPORT_PLUGIN(qkrcodecs)
-Q_IMPORT_PLUGIN(qtwcodecs)
 #endif
 
 #ifdef Q_OS_WIN32
--- phantomjs-1.9.7/./src/phantomjs.pro~	2014-03-02 16:26:44.000000000 +0200
+++ phantomjs-1.9.7/./src/phantomjs.pro	2014-03-02 16:28:02.520192429 +0200
@@ -58,14 +58,6 @@
 LIBS += -llinenoise
 LIBS += -lqcommandline
 
-linux*|mac {
-    QTPLUGIN += \
-        qcncodecs \
-        qjpcodecs \
-        qkrcodecs \
-        qtwcodecs
-}
-
 win32: RC_FILE = phantomjs_win.rc
 os2:   RC_FILE = phantomjs_os2.rc
 
@@ -82,10 +74,6 @@
     CONFIG(static) {
         DEFINES += STATIC_BUILD
         QTPLUGIN += \
-            qcncodecs \
-            qjpcodecs \
-            qkrcodecs \
-            qtwcodecs \
             qico
     }
 }
